profile/CSIP.xml

```diff
@@ -154,7 +154,7 @@
             <requirement ID="CSIP1" REQLEVEL="MUST" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
                     <head>Content Identification</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">It is mandatory to use a content ID which is expressed with @OBJID. The value should be the same as the name or ID of the package (the name of the root folder) for the root METS document or the name and folder name for the representation. The OBJID must meet the principle of being unique at least across the repository.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">It is mandatory to use a content identification in the form of a string which is expressed within the attribute @OBJID. The value should be the same as the name or ID of the package (the name of the root folder) for the root METS document or the name and folder name for the representation. The OBJID must meet the principle of being unique at least across the repository.</p>
```

Comment:
It is mandatory to use the @OBJID attribute as a string identifier for the package. The value should be identical to the name/ID of the package, i.e. the name of the root folder, for the root METS document or the name and folder name for a representation.

Reply:
Fixed.